host/src/components/ModuleNavigator.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from "react";
```

Comment:
This is a new component to handle the Design team updates for the footer. Because the `host` transition design relies on a bunch of components stacked up (confidence, real time responses, game answers, for instance) the DEsign team has requested a navigation bar with a drop down and buttons to be able to scroll between these components easily. This component accomplishes that (and it is refined in subsequent PRs).